### PR TITLE
Feature/1457 add shorcuts for pause breaks

### PR DESCRIPTION
**Pull request by evgenyneu**
Issue: #1457

### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.

- [x]  issue was opened to discuss proposed changes before starting implementation. It is important do discuss changes before implementing them (Why should we add it? How should it work? How should it look? Where will it be? ...).
- [x]  during development, `node` version specified in `package.json` was used (ie using [nvm](https://github.com/creationix/nvm)).
- [x]  package versions and package-lock.json were not changed (`npm install --no-save`).
- [x]  app version number was not changed.
- [x]  all new code has tests to ensure against regressions.
- [x] `npm run lint` reports no offenses.
- [x] `npm run test` is error-free.
- [x]  README and CHANGELOG were updated accordingly.
- []  after PR is approved, all commits in it are [squashed](https://gitbetter.substack.com/p/how-to-squash-git-commits)

### Description of the Change

* Added ability to set keyboard shortcuts to pause the breaks for the period of time (30 min, 1 hr etc.)
* Moved the code for keyboard chortcut registration from `main.js` into `utils/breakShortcuts.js`.

### Verification Process

* Open About tab in the Preferences screen.
* Click Ctrl-D to see the path to preferences JSON
* Set the keyboard shortcuts:

```
"pauseBreaksFor30MinutesShortcut": "CmdOrCtrl+Alt+0",
"pauseBreaksFor1HourShortcut": "CmdOrCtrl+Alt+1",
"pauseBreaksFor2HoursShortcut": "CmdOrCtrl+Alt+2",
"pauseBreaksFor5HoursShortcut": "CmdOrCtrl+Alt+5",
"pauseBreaksUntilMorningShortcut": "CmdOrCtrl+Alt+M",
```

* Restart the app.
* Use the keyboard shortcuts and see that the breaks are paused: the app icon changes to Pause and when you click the text shows the correct pause duration.

* Since I moved the code for other break shortcuts into, I also tested these shortcuts:
```
"pauseBreaksToggleShortcut": "CmdOrCtrl+Alt+T",
"skipToNextScheduledBreakShortcut": "CmdOrCtrl+Alt+B",
"skipToNextMiniBreakShortcut": "CmdOrCtrl+Alt+S",
"skipToNextLongBreakShortcut": "CmdOrCtrl+Alt+L",
"resetBreaksShortcut": "CmdOrCtrl+Alt+R",
```

* I tested it on Mac, Windows and Ubuntu.

### CODE REVIEW

File: app/main.js [hunk 1]
```diff
@@ -1315,26 +1317,31 @@ function getTrayMenuTemplate () {
       submenu: [
         {
           label: i18next.t('utils.minutes', { count: 30 }),
+          accelerator: settings.get('pauseBreaksFor30MinutesShortcut'),
```
Comment (hovancik) [inline app/main.js]: I guess not that important but returns `[18960:0723/192752.999383:WARNING:accelerator_util.cc(65)]  doesn't contain a valid key` when no value set.

File: app/utils/pauseBreaksShortcut.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+const { UntilMorning } = require('./untilMorning')
```
Comment (hovancik) [inline app/utils/pauseBreaksShortcut.js]: Could you please what do you think about registering `pauseBreaksToggleShortcut`? Haven't had time think myself, but it hits me we have one shortcut left in `main.js` regarding to pausing.

Or even put all shortcuts here? (like skipToNextScheduledBreakShortcut, skipToNextMiniBreakShortcut and such).

**Comment by hovancik** [comment]
I've made some comments, let me know what you think :) 

I can test then on Windows as well.

File: app/main.js (hunk 1 above)
Comment (evgenyneu) [reply app/main.js]: Well spotted. How do I reproduce this error? I tried to start the app (`npm run start`) with empty string `""` values for the shortcuts but I don't see this error in the terminal.

Comment (hovancik) [reply app/main.js]: Try with `npm run dev`, it gives more debug info.

Comment (evgenyneu) [reply app/main.js]: Fixed, the warning only appeared in Ubuntu for me but not in Windows or Mac.

File: app/utils/pauseBreaksShortcut.js (hunk 2 above)
Comment (evgenyneu) [reply app/utils/pauseBreaksShortcut.js]: Done

**Comment by evgenyneu** [comment]
I've made the improvements you suggested. Also, I managed to install the dev environment on Windows, and tested the shortcuts on all platforms. Let me know if you want to change/improve anything. :D

**Review by hovancik — APPROVED** [review]
Many thanks, looks good :)

**Comment by hovancik** [comment]
I can squash and merge, unless you want to add yourself to the readme :)

**Comment by evgenyneu** [comment]
Thanks, squash is ok, I don't need readme mention :D

**Pull request merged**